content/md/first-steps/manage-your-attributes.md

```diff
@@ -7,141 +7,96 @@ ee-only: false
-For example, you can choose to create a text attribute (free text) or a drop down list (simple or multi select) with a selection of values, to make sure users will select one of the predefined options. Create a `metric` attribute to fill in the weight/size/length... of your products.
-
-To create an attribute in the PIM:
+To create an attribute in Akeneo PIM:
 1. Go to `Settings/Attributes`
-1. Click on the `Create attribute` button
+1. Click on the `Create attribute` button:
   ![image](../img/Settings_AttributesCTAButton.png)
-1. Select the attribute type you want to create
+1. Select the attribute type you want:
   ![image](../img/Settings_Attributes_Create2.png)
-1. The attribute edit page is displayed. The following properties are required to finalise the creation of your attribute:
-    - A **code** to identify the attribute
-    - An **attribute group**: the attribute group the attribute will belong to
-    - A **unique value** (Yes/No): if the value for this attribute is unique (field available or required depending on the attribute type)
-    - A **value per channel** (Yes/No): Will your product attribute values differ on the channels?
-    - A **value per locale** (Yes/No): Will your product attribute values differ on the locales?
-    - **Locale specific**: to enable this attribute only to specific locales and not all of them
-    - **Usable in grid**: whether or not the attribute can be displayed as a column or used as a filter in the product grid _(Community and Enterprise Edition)_ or in the proposal grid _(Enterprise Edition only)_
-    - **Read only**: when enabled it prevents this attribute from being edited through the UI, it can only be updated through imports, API or rules _(Enterprise Edition only)_
-    - **Validation parameters**: depending on the attribute type chosen (see [Validation for the different attribute types](#parameters-according-to-specific-attribute-types)).    
+1. The attribute edit form is displayed, fill in the following properties:  
+    Under `General Parameters`: 
+    - A **code** (required) to identify the attribute.
+    - A **type** based on the attribute type selected in the previous screen.
+    - An **attribute group** (required) in which the new attribute will be added to.
+    In addition to these required attributes, you can select the following properties:
+    - **Unique value** (Yes/No): enable this property if the value of this attribute must be unique. By default, it is set to yes for `Identifier` attribute type.
+    - **Value per channel** (Yes/No): enable this property if you plan to have different product values per channel.
+    - **Value per locale** (Yes/No): enable this property if you plan to have different product values per locale.
+    - **Read only** (Yes/No): to prevent this attribute from being edited through the UI, it will only be updated through imports, API or rules _(Enterprise Edition only)_
```

Comment:
```suggestion
    - **Read only** (Yes/No) **_(Enterprise Edition only)_**: to prevent this attribute from being edited through the UI, it will only be updated through imports, API or rules
```